website/docs/docs/cloud/about-cloud/browsers.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Supported browsers"
+id: "browsers"
+description: "dbt Cloud supports the latest browsers like Chrome and Firefox."
+---
+
+To have the best experience with dbt Cloud, we recommend using the latest versions of the following browsers:
+
+- [Google Chrome](https://www.google.com/chrome/) 
+- [Mozilla Firefox](https://www.mozilla.org/en-US/firefox/)
+
+If you don't use the latest version of a recommended browser or if you use a browser that isn't listed, some dbt Cloud features may not work as expected.
+
+Note: To improve your experience using dbt Cloud, we suggest that you turn off ad blockers.
```

Comment:
i added this since this is recommended for the ide [see here](https://docs.getdbt.com/docs/cloud/dbt-cloud-ide/develop-in-the-cloud)